main_routers/music_router.py

```
@@ -71,6 +107,11 @@ async def play_netease_music(song_id: str):
     网易云 VIP 音乐智能跳转路由：
     利用后端 MUSIC_U Cookie 获取真实高音质/鉴权直链，通过 307 重定向至前端播放。
     """
+    if not _PYNCM_AVAILABLE:
+        fallback_url = f"https://music.163.com/song/media/outer/url?id={song_id}.mp3"
+        logger.warning(f"[音乐播放] pyncm_async 不可用，直接使用公开外链")
```

Ruff (0.15.9)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)